### DISCUSSION TITLE
[REQUEST] is support pyspark

### DISCUSSION THREAD

**Discussion by Peterpig** [Textualize/rich#2656] [General]
Consider posting in https://github.com/textualize/rich/discussions for feedback before raising a feature request.

Have you checked the issues for a similar suggestions?

**How would you improve Rich?**

```
from pyspark.sql import SparkSession
from rich.logging import RichHandler

spark = SparkSession()
log4j_logger = spark._jvm.org.apache.log4j

log = log4j_logger.LogManager.getLogger('xx')

handler = RichHandler()
log.addHandler(richhandle)

```

**What problem does it solve for you?**

![image](https://user-images.githubusercontent.com/7956973/202345415-411fd025-5732-4679-bea0-f7b94f1758a3.png)

it raise AttributeError

What problem do you have that this feature would solve? I may be able to suggest an existing way of solving it.

pyspark log4j logger with  ritch

**Comment by willmcgugan** [comment]
This appears to be an issue with `py4j`. You might want to post an issue there.
